pyccel/codegen/compiling/compilers.py

```diff
@@ -39,7 +39,12 @@ def get_condaless_search_path():
                           'Conda', 'Anaconda', 'Miniconda')
     conda_folders = [p for p,f in folders.items() if any(con in f for con in conda_folder_names)]
     if conda_folders:
-        warnings.warn(UserWarning("Ignoring conda paths when searching for compiler : {}".format(conda_folders)))
+        if not is_conda_warnings_disabled:
+            message_warning = "Conda paths are ignored. See https://github.com/pyccel/pyccel/blob/devel/tutorial/compiler.md for details"
```

Comment:
I think you can provide a better URL:
https://github.com/pyccel/pyccel/blob/devel/tutorial/compiler.md#utilising-pyccel-within-a-conda-environment